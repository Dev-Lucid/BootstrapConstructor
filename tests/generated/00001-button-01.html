<button class="btn">button-01</button>