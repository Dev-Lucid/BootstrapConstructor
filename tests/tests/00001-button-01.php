<?php
$button = bsc::button',array('label'=>'button-01());
file_put_contents($output_path,$button->render());
?>