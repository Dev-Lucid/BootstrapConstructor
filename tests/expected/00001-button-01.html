<a class="btn">button-01</a>